code/modules/integrated_electronics/core/printer.dm

```diff
@@ -192,7 +192,7 @@
 		else if(ispath(build_type, /obj/item/integrated_circuit))
 			var/obj/item/integrated_circuit/IC = SScircuit.cached_components[build_type]
 			cost = IC.materials[MAT_METAL]
-		else
+		else if(!ispath(build_type, /obj/item/integrated_electronics))
```

Comment:
This is literally a reversal of the previous check and reopens the original issue. Just three lines above

Reply:
See http://www.byond.com/docs/ref/info.html#/proc/ispath
This check above should be letting people make the wirer. If it isnt then something else is the issue instead of it right here.

Reply:
>This is literally a reversal of the previous check (Three lines above) and reopens the original issue.

The one above says /obj/item/integrated circuit. Which is the path for the circuits. This, on the other hand, is /obj/item/integrated_electronics, which is the path that all of the tools are under. Why you think one would allow the other, I don't know.